src/Installers/Windows/SharedFrameworkBundle/Bundle.wxs

```diff
@@ -3,6 +3,7 @@
     <Bundle Name="$(var.BundleName)" Version="$(var.BundleVersion)" Manufacturer="Microsoft Corporation" UpgradeCode="$(var.BundleUpgradeCode)"
             dep:ProviderKey="$(var.BundleProviderKey)">
         <BootstrapperApplicationRef Id="WixStandardBootstrapperApplication.HyperlinkLicense">
+            <bal:WixStandardBootstrapperApplication SuppressDowngradeFailure="yes"/>
```

Comment:
Can you just add the attribute to the existing element? We already have the WixStandardBootstrapperApplication entry

Reply:
Whoops, good call - done